roles/falcon_install/tasks/win_install.yml

```diff
@@ -1,38 +1,37 @@
 ---
-- block:
-  - name: CrowdStrike Falcon | Install Falcon Sensor .exe Package (Windows)
-    ansible.windows.win_package:
-      path: "{{ falcon_sensor_pkg }}"
-      state: present
-      creates_service: csfalconservice
-      arguments: '/install /quiet CID={{ falcon_cid }} {{ falcon_windows_install_args }}'
-    when:
-      - ansible_os_family == "Windows"
-    register: falcon_installed
-    retries: "{{ falcon_windows_install_retries }}"
-    delay: "{{ falcon_windows_install_delay }}"
-    until: falcon_installed is success
+- name: CrowdStrike Falcon | Install Falcon Sensor .exe Package (Windows)
+  ansible.windows.win_package:
+    path: "{{ falcon_sensor_pkg }}"
+    state: present
+    creates_service: csfalconservice
+    arguments: '/install /quiet CID={{ falcon_cid }} {{ falcon_windows_install_args }}'
+  when:
+    - ansible_os_family == "Windows"
+  register: falcon_installed
+  retries: "{{ falcon_windows_install_retries }}"
+  delay: "{{ falcon_windows_install_delay }}"
```

Comment:
Should the quotes be removed since it makes it a string and not an int?

Reply:
That would cause a YAML Gotcha error since anything that starts with {{ must be quoted